client/extensions/woocommerce/woocommerce-services/state/shipping-label/reducer.js

```diff
@@ -477,13 +474,13 @@ reducers[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SET_PACKAGE_TYPE ] = ( state, { pa
 	const box = state.form.packages.all[ boxTypeId ];
 	const weight = round(
```

Comment:
This was the previously local `round` which encapsulated the precision, so needs an adjustment here as well